### PR TITLE
Feature/add comment buttons

### LINKED ISSUE #115: style comment buttons
These buttons will allow the user to like, dislike and/or remove a specific comment

### LINKED ISSUE #116: Add comment buttons
These buttons will allow the user to like, dislike and/or remove a specific comment

### PR DESCRIPTION
**Pull request by Atos20**
#### What’s this PR do?  
- it styles the buttons to interact with each user comment
- it adds like, dislike, and remove buttons
#### Where should the reviewer start?  
 - `comment.js` `comment.scss`
#### How should this be manually tested?  
 run `npm start`
- Click on a movie card 
- head to the comments sections
#### Any background context you want to provide?  
 this PR is part of the iteration `7a` 
#### What are the relevant tickets?  
closes #115 
closes #116 
#### Screenshots (if appropriate) 
  
Screen Shot 2020-10-18 at 1 23 35 PM

### CODE REVIEW

File: src/Comments/Comments.js [hunk 1]
```diff
@@ -55,15 +56,24 @@ export class Comments extends Component  {
...
               </button>
 
+              <button 
+              role="clear-button"
+              type="clear-button"
+              onClick={this.clearForm}
+              name="comment"
+              className="clear-button">
+              clear
+              </button>
```
Comment (holladayian) [inline src/Comments/Comments.js:75]: Good thinking adding this! It definitely adds to the UI

File: src/Comment/Comment.js [hunk 2]
```diff
@@ -8,17 +8,21 @@ export const Comment = (props) => {
...
         <h1 className="user">{props.comment.author}</h1> 
         <h1 className="time">3 months ago</h1> 
       </div>
+
      <p className="comment-text">{props.comment.comment}</p>
+     
      <div className="buttons-container">
        <i className="far fa-thumbs-up"></i>
-       <h3 className="like-count">1</h3>
+       <h3 className="like-count">0</h3>
```
Comment (holladayian) [inline src/Comment/Comment.js:21]: Is this for the ability for people to like each other's comments?

Comment (Atos20) [reply src/Comment/Comment.js:21]: that is correct 👍

File: src/Comments/Comments.js (hunk 1 above)
Comment (Atos20) [reply src/Comments/Comments.js:75]: nice!

**Pull request merged**